Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.7.4"
 
 [deps]
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+SafeTestsets = "1bc83da4-3b8d-516f-aca4-4fe02f6d838f"
```

Comment:
Let's move this to the `test/` environment, CLIMAParameters doesn't directly depend on `SafeTestsets`.